src/main/java/org/takes/facets/auth/codecs/CcBase64.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.facets.auth.codecs;
+
+import java.io.IOException;
+import lombok.EqualsAndHashCode;
+import org.takes.facets.auth.Identity;
+
+/**
+ * Base64 codec.
+ *
+ * <p>The class is immutable and thread-safe.
+ *
+ * @author Igor Khvostenkov (ikhvostenkov@gmail.com)
+ * @version $Id$
+ * @since 0.13
+ */
+@EqualsAndHashCode
+public final class CcBase64 implements Codec {
+    /**
+     * Original codec.
+     */
+    private final transient Codec origin;
+    /**
+     * Ctor.
+     * @param codec Original codec
+     */
+    public CcBase64(final Codec codec) {
+        this.origin = codec;
+    }
+
+    /**
+     * This is @todo #19 to implement own simple Base64 encode algorithm
```

Comment:
@ikhvostenkov that's because the `@todo` should come AFTER the javadoc text and after the `@param` entries. As a reference, look at this puzzle from `PsLinkedinTest`. Note the `@todo` comes last. Also, look at the `#12:30min` part. yours should read `#19:30min`. The `/DEV` part is optional.
```
    /**
     * PsLinkedin can login.
     * @throws IOException If some problem inside
     * @todo #12:30min/DEV This class not implemented yet, but has to be.
     *  Please implement it. Mocked OAuth server is one of the possible
     *  solutions. In this case, we could use some library like WireMock
     *  (see http://www.wiremock.org), which allows make stub for web
     *  service. We'll need to make some changes in PsLinkedin,
     *  that allow to change API url, and use local stubbed http
     *  server during unit test.
     */
```